src/overview/tabs/Details/cards/WelcomeCard.tsx

```diff
@@ -0,0 +1,158 @@
+import { type FC, type Ref, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router-dom-v5-compat';
+import migrationIcon from 'src/components/images/resources/migration.svg';
+import ovaIcon from 'src/components/images/resources/open-virtual-appliance.png';
+import openShiftVirtualizationIcon from 'src/components/images/resources/openshift-virtualization.svg';
+import openStackIcon from 'src/components/images/resources/openstack2.svg';
+import redHatIcon from 'src/components/images/resources/redhat.svg';
+import vmwareIcon from 'src/components/images/resources/vmware-dark.svg';
+import vmwareIconLight from 'src/components/images/resources/vmware-light.svg';
+import providerTypes from 'src/modules/Plans/views/create/constanats/providerTypes';
+import { getResourceUrl } from 'src/modules/Providers/utils/helpers/getResourceUrl';
+import HideFromViewDropdownOption from 'src/overview/components/HideFromViewDropdownOption';
+import { useIsDarkTheme } from 'src/utils/hooks/useIsDarkTheme';
+import { ForkliftTrans, useForkliftTranslation } from 'src/utils/i18n';
+
+import { ProviderModelRef, type V1beta1ForkliftController } from '@kubev2v/types';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+  Dropdown,
+  DropdownList,
+  MenuToggle,
+  type MenuToggleElement,
+  Split,
+  SplitItem,
+  Text,
+  Tile,
+} from '@patternfly/react-core';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+
+type WelcomeCardProps = {
+  obj?: V1beta1ForkliftController;
+  loaded?: boolean;
+  loadError?: unknown;
+  onHide: () => void;
+};
+
+const WelcomeCard: FC<WelcomeCardProps> = ({ onHide }) => {
+  const { t } = useForkliftTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  const navigate = useNavigate();
+  const isDarkTheme = useIsDarkTheme();
+
+  const providersListUrl = useMemo(() => {
+    return getResourceUrl({
+      namespace: activeNamespace,
+      namespaced: true,
+      reference: ProviderModelRef,
+    });
+  }, [activeNamespace]);
+  const providersCreateUrl = `${providersListUrl}/~new`;
+  const [menuIsOpen, setMenuIsOpen] = useState(false);
+  const actionDropdownItems = [<HideFromViewDropdownOption onHide={onHide} />];
+  const onToggle = () => {
+    setMenuIsOpen((open) => !open);
+  };
+
+  const headerActions = (
+    <Dropdown
+      isOpen={menuIsOpen}
+      onOpenChange={(isOpen: boolean) => {
+        setMenuIsOpen(isOpen);
+      }}
+      toggle={(toggleRef: Ref<MenuToggleElement>) => (
+        <MenuToggle ref={toggleRef} onClick={onToggle} isExpanded={menuIsOpen} variant={'plain'}>
+          {<EllipsisVIcon />}
+        </MenuToggle>
+      )}
+      shouldFocusFirstItemOnOpen={false}
+      popperProps={{
+        position: 'right',
+      }}
+    >
+      <DropdownList>{actionDropdownItems}</DropdownList>
+    </Dropdown>
+  );
+
+  type ProviderType = keyof typeof providerTypes;
```

Comment:
you added a type inside a component, please extract it form the component function